### ISSUE TITLE
Hover effect 

### ISSUE THREAD

**Issue by kumarishalini6** [Girl-Code-It/Opportunity-Calendar-Frontend#249]
### **Feature request**:

There should be some hover effect in opportunity page's boxes, i want to work on this issue under GSSoC'21.
![Opportunity Calendar - Personal - Microsoft​ Edge 5_8_2021 06_43_52 PM](https://user-images.githubusercontent.com/69043944/117572210-48e0ec00-b07e-11eb-9cee-f58bdb933db0.png)

**Comment by vaishali614** [comment]
@kumarishalini6 what kind of hover effect do you wanna add?

**Comment by kumarishalini6** [comment]
I want to give some border shadow , little amount of gray color inside box , and hover effect as in home page's opportunity section

**Comment by kumarishalini6** [comment]
if you don't like the idea you can close the issue , i will create new issue.

**Comment by vaishali614** [comment]
@kumarishalini6 Let's keep it similar to the hover effect that's on homepage. Also, please take the latest pull from upstream before starting and before making a PR.

**Comment by kumarishalini6** [comment]
Ok

**Comment by kumarishalini6** [comment]
I have send PR with  latest upstream . please check it.

**Comment by kumarishalini6** [comment]
Please close this issue

**Comment by vaishali614** [comment]
> Please close this issue

You should ideally mention the issue with your PR, so that when the PR is merged , the issue closes automatically.
